build.gradle

```diff
@@ -59,12 +61,7 @@ repositories {
     jcenter()
 }
 
-ext {
-    opendistroVersion = '1.0.0'
-    isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-}
-
-version = "${opendistroVersion}.0"
+version = "1.0.0"
```

Comment:
Minor nit, please include last digit for minor version updates.

Reply:
Actually not so minor, thank you! I would have made a mess of versions across 3 repos. Much better now.